tfhe/src/core_crypto/entities/modulus_switched_lwe_ciphertext.rs

```diff
@@ -187,10 +195,20 @@ impl<Scalar: UnsignedTorus> PackedModulusSwitchedLweCiphertext<Scalar> {
                     //                                            |---|
                     //                                       start_remainder
                     //
-                    // container[i] = log_modulus lowest_bits of
-                    self.packed_coeffs[start_block] >> start_remainder
+                    // In container[i] we wants the bits of packed_coeffs[start_block] starting from
+                    // index start_remainder
+                    //
+                    // container[i] = lowest_bits of single_part
+                    //
+                    // The highest bits of single_part will be discared during scaling
+                    let single_part = self.packed_coeffs[start_block] >> start_remainder;
+
+                    #[allow(clippy::let_and_return)]
+                    single_part
```

Comment:
I think it's fine to have
// single_part = 
self.packed_coeffs[start_block] >> start_remainder

and avoid the clippy thing